tests/baselines/reference/jsFileCompilationDuplicateVariableErrorReported.js

```diff
@@ -14,4 +14,4 @@ var x = 10; // Error reported
 
 //// [out.d.ts]
 declare var x: string;
-declare var x: string;

```

Comment:
...? Feels like we need to talk about the behavior of multiple declarations of the same thing :(

I don't think the result determined by the syntactic printer for the second declaration (`number`) jives with the inferred type of the declaration (`string`). When a value has multiple declarations, I think it's only safe to apply the node reuse logic on the first one.

Reply:
Isn't this always an error case (this case is an error case). If so is it really a problem that the type gets printed is different?

Reply:
In the context of `//@ts-ignore` existing at all, yes - it might not need to be ID-compatible, since if you suppress an ID error all bets are off with respect to the output being ID-similar, but we still want accurate-to-the-best-of-our-analysis declarations for incremental compilation.

Reply:
Reverted to old emit.